javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplateBase.java

```diff
@@ -205,17 +206,34 @@ private static void bodySchemaJavadoc(IType type, JavaJavadocComment commentBloc
             } else {
                 commentBlock.line(indent + appendOptionalOrRequiredAttribute(isRequired, isRootSchema) + "{");
             }
-            bodySchemaJavadoc(((MapType) type).getValueType(), commentBlock, nextIndent, "String", typesInJavadoc, isRequired, false);
+            bodySchemaJavadoc(((MapType) type).getValueType(), commentBlock, nextIndent, "String", typesInJavadoc, isRequired, false, true);
             commentBlock.line(indent + "}");
         } else {
+            String javadoc;
+            if (isInRequestBody) {
```

Comment:
In DPG, when response has binary schema, response type for the generated proxy method is `Response<Void>`. I believe there's no `Flux<ByteBuffer>` for response body type (which shouldn't be converted anyway), but for clarity, I restrict the conversion only in request body schema.

Reply:
Basically, for DPG, if response is not Void, it would be `BinaryData`. So it is fine to convert `Flux<ByteBuffer>` to `BinaryData`  in response (if it happens).

Reply:
However, above should actually have `BinaryData` as response, not `Void`.

Do you have the source?

@haolingdong-msft for possible investigation (this would be another issue not related to this PR)

Reply:
At least make these be good
https://github.com/Azure/autorest.testserver/blob/main/swagger/body-file.json

Reply:
I manually added the above swagger content to test the result. For fluent, response type will be `StreamResponse` and `Flux<ByteBuffer>`, for DPG, it will be `Response<Void>`.

Reply:
I can help with the investigation if haoling has other priorities.

Reply:
> I manually added the above swagger content to test the result. For fluent, response type will be `StreamResponse` and `Flux<ByteBuffer>`, for DPG, it will be `Response<Void>`.

The DPG likely be wrong. (what if 200 is before 204 in swagger?)

Reply:
Same `Response<Void>` if 200 is before 204.